test/mockDGXDEX.js

```diff
@@ -1,11 +1,11 @@
-var TestToken = artifacts.require("./mockContracts/TestToken.sol");
-var mockDGXDEX = artifacts.require("./mockContracts/MockDGXDEX.sol")
-var Helper = require("./helper.js")
+let TestToken = artifacts.require("./mockContracts/TestToken.sol");
```

Comment:
I think here const is better.
all these values will not change.

Reply:
ok, but note that searching "const" smart contracts in test dir found only one occurence (and lots of "var" :)). So you don't really use it..